### PR TITLE
GP Internal Core API v1.1 : review internal client and cancellation func...

### PR DESCRIPTION
**Pull request by cedric-chaumont-st-dev**
...tions

update panic reasons
checkpatch warning fix (else after return)

Signed-off-by: Cedric Chaumont <cedric.chaumont@st.com>

### CODE REVIEW

File: lib/libutee/tee_api.c [hunk 1]
```diff
@@ -78,9 +78,16 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
 			       uint32_t commandID, uint32_t paramTypes,
 			       TEE_Param params[4], uint32_t *returnOrigin)
 {
-	return utee_invoke_ta_command(session, cancellationRequestTimeout,
+	TEE_Result res;
+	res = utee_invoke_ta_command(session, cancellationRequestTimeout,
 				      commandID, paramTypes, params,
 				      returnOrigin);
+	if (res != TEE_SUCCESS &&
+		res != TEE_ERROR_OUT_OF_MEMORY &&
```
Comment (pascal-brand-st-dev) [inline lib/libutee/tee_api.c]: Indentation: align the "res" in the test

**Comment by pascal-brand-st-dev** [comment]
A relevant travis error is there, and the commit message is malformed.

File: lib/libutee/tee_api.c [hunk 2]
```diff
@@ -78,9 +78,16 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
 			       uint32_t commandID, uint32_t paramTypes,
 			       TEE_Param params[4], uint32_t *returnOrigin)
 {
-	return utee_invoke_ta_command(session, cancellationRequestTimeout,
+	TEE_Result res;
+	res = utee_invoke_ta_command(session, cancellationRequestTimeout,
```
Comment (jenswi-linaro) [inline lib/libutee/tee_api.c]: Please add an empty line between local variables and code.

**Comment by cedric-chaumont-st-dev** [comment]
Travis fix.

**Comment by pascal-brand-st-dev** [comment]
Once the alignement on a "if" is fixed:

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

File: lib/libutee/tee_api.c [hunk 3]
```diff
@@ -88,30 +96,35 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
...
-		TEE_Panic(res);
+		c = false;
+
 	return c;
 }
 
 bool TEE_UnmaskCancellation(void)
 {
 	bool old_mask;
+
```
Comment (jenswi-linaro) [inline lib/libutee/tee_api.c]: Empty line not needed.

File: lib/libutee/tee_api.c [hunk 4]
```diff
@@ -88,30 +96,35 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
...
 	if (res != TEE_SUCCESS)
 		TEE_Panic(res);
+
 	return old_mask;
 }
 
 bool TEE_MaskCancellation(void)
 {
 	bool old_mask;
+
```
Comment (jenswi-linaro) [inline lib/libutee/tee_api.c]: Empty line not needed.

File: lib/libutee/tee_api.c [hunk 5]
```diff
@@ -88,30 +96,35 @@ TEE_Result TEE_InvokeTACommand(TEE_TASessionHandle session,
 bool TEE_GetCancellationFlag(void)
 {
 	bool c;
-	TEE_Result res = utee_get_cancellation_flag(&c);
 
+	TEE_Result res = utee_get_cancellation_flag(&c);
```
Comment (jenswi-linaro) [inline lib/libutee/tee_api.c]: Please remove the empty line above this line and add an empty line below.

**Comment by cedric-chaumont-st-dev** [comment]
Updated

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Pull request merged**